src/Components/Tasks/Tasks.tsx

```diff
@@ -76,24 +81,58 @@ const Tasks: React.FC = () => {
         getToastValue(
           "error",
           error?.response?.data?.message ||
-            "An error occurred. Please try again."
+          "An error occurred. Please try again."
         );
       });
   };
+  // **********get all Employee tasks**********pageSize:number, pageNumber:number*******
+  const getEmployeeTasksList = async () => {
```

Comment:
I noticed that you are mixing between the async await and then catch syntax.
You can replace the then catch with async await for better readability as follows

```
const fetchData = async () => {
  try {
    const response = await axios.get(`${baseUrl}/Task`, {
      headers: requestHeaders,
  });

    const { data } = response;
    
    ...... other code

  } catch (error) {
    // Handle errors
    getToastValue(
      'error',
      error?.response?.data?.message || 'An error occurred. Please try again.'
    );
  }
};

```